src/core/handle.ml

```diff
@@ -187,6 +187,7 @@ let handle_cmd : sig_state -> p_command -> sig_state * proof_data option =
       let d = if op then None else Some(t) in
       let ss = Sig_state.add_symbol ss e Defin x a impl d in
       (ss, None)
+  | P_inductive(_, _, _, _) -> failwith "to be implemented" (* P_inductive(e, s, t, tl) @Moi *)
```

Comment:
Please, to be use failwith. Instead, just make a warning using wrn (see example at the end of handle.ml).

Reply:
It's OK now.